guides/plugins/apps/administration/add-custom-modules.md

```diff
@@ -141,3 +141,10 @@ To avoid mixing other modules with your main module, we decided to separate the
 
 This feature is not compatible with themes as they will always open the theme config by default.
 
+## Admin design compatibility
```

Comment:
We are aware of the difficulties of getting the look&feel of your iframes to match the rest of the administratrion, we have a solution in mind, but are not sure when we can deliver on that. For the meantime i am fine with this creative approach. @Christian-Rades what do you think?

Reply:
Agreed, I think it's a good Idea :+1:

Reply:
"Creative approach" :) I don't see where programming isn't creative